tailwind-theme-preset.js

```diff
@@ -318,6 +318,10 @@ module.exports = {
       '2xl': '1600px',
     },
     extend: {
+      aspectRatio: {
```

Comment:
You don't have to define those aspect ratio, the theme already have configured all of them. See line 7

We might need to add a wrapper to the image and set the `aspect ratio` when we use `next/image`, so we might need to do some testing on implementing this in Next.js